src/users/users.service.ts

```diff
@@ -8,6 +8,7 @@ import { PrismaService } from '../prisma/prisma.service';
 import { User, Prisma } from '@prisma/client';
 import { CreateUserDto } from './dto/create-user.dto';
 import { UpdateUserDto } from './dto/update-user.dto';
+import { encodePassword } from 'src/utils/bcrypt';
```

Comment:
**praise**: Encoding the password before storing it is a crucial security measure.